kimchi/src/prover_index.rs

```diff
@@ -91,6 +100,45 @@ where
             max_poly_size,
             max_quot_size,
             fq_sponge_params,
+            verifier_index: None,
+            verifier_index_digest: None,
+        }
+    }
+
+    /// Retrieve or compute the digest for the corresponding verifier index.
+    /// If the digest is not already cached inside the index, store it.
+    pub fn compute_verifier_index_digest<
```

Comment:
this function is unused

Reply:
looks like it's used on the ocaml side

Reply:
We don't always have a mutable reference -- in particular, in the kimchi prover, the reference is immutable, and I don't think we want to change it to be mutable. This extra complexity is to deal with that.